Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/rating/RatingRepo.kt

```diff
@@ -0,0 +1,40 @@
+package com.mentalmachines.droidcon_boston.views.rating
+
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.ValueEventListener
+import timber.log.Timber
+
+class RatingRepo(
+    private val userId: String,
+    private val userDatabase: DatabaseReference
+) {
+    fun getSessionFeedback(sessionId: String, feedbackCallback: (SessionFeedback?) -> Unit) {
+        userDatabase.child("$userId/sessionFeedback/$sessionId")
```

Comment:
perhaps we could pull out a constant for this path?  not sure if that will totally work because of the templating.  maybe if it's on a companion and has access to the properties 🤔

Reply:
hmm...good point...I've never tried a constant for string interpolation....

Reply:
couldn't pull it into a constant...had to be a member function to get access to the user ID unfortunately :-(